README.md

```diff
@@ -62,6 +62,14 @@ We use [Discord](https://discordapp.com/) for community discussion. You can use
 
 Notion is intended to compile with all versions of Rust newer than 1.20.
 
+## Tests
+
+To run the tests in this repo, run the following:
+
+```
+cargo test --features mock-network
```

Comment:
```suggestion
cargo test --all --features mock-network
```

Reply:
:angel: trying out the new feature.

Reply:
Whoops, they didn't send me a notification for the suggested fix 😕 and I clobbered it...

Reply:
Oh that's an interesting new thing!